docs/addons/install-addons.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Set up Storybook addons
+---
+
+Storybook boasts a wide variety of addons in it's ecosystem. All of them packaged as NPM modules and listed in our [addon gallery](/addons). We're going to guide you through the necessary steps to install and register Storybook addons.
+
+### Install Storybook addons
+
+With the exception of presets, all other Storybook addons follow the same set of installation instructions. For example, to include accessibility testing in Storybook, run the following command to install the necessary addon:
+
+```shell
+yarn add -D @storybook/addon-a11y
+```
+
+### Addon registration
+
+Installing the addon is not enough, it needs to be registered. Update [`.storybook/main.js`](../configure/overview.md#configure-story-rendering) to the following:
```

Comment:
Minor language recommendation. Saying
> Installing the addon is not enough

makes the usage process sound more laborious. Could you reword this to something like:
> ### Using addons
> With the exception of presets, all other Storybook addons follow the same set of installation instructions. It's a 2 step process: install and register. 
>
> For example, to include accessibility testing in Storybook, run the following command to install the necessary addon:
>...

Reply:
Updated with the latest commit